metaverse_tools/ui/armature_tools.py

```diff
@@ -108,6 +105,53 @@ def execute(self, context):
         bpy.context.object.data.use_mirror_x = use_mirror_x
         return {'FINISHED'}
 
+
+class BONES_OT_MVT_TOOLSET_Rename_Full_Chain(bpy.types.Operator):
+    """ Rename bones to Chain From Root Name and Numbers Accordingly """
+    bl_idname = "metaverse_toolset.rename_full_bone_chain"
+    bl_label = "Rename Chain"
+
+    bl_region_type = "TOOLS"
+    bl_space_type = "VIEW_3D"
+    bl_options = {'REGISTER', 'UNDO', 'INTERNAL'}
+
+    target_name: bpy.props.StringProperty()
+    different_name: bpy.props.BoolProperty(default=False)
+
+    @classmethod
+    def poll(self, context):
+        return context.selected_bones is not None and len(context.selected_bones) >= 1
```

Comment:
Just making sure this is a correct expression. :)

Reply:
I didn't look at this file yet, I will test if it works